privileges/privileges_win.go

```diff
@@ -0,0 +1,8 @@
+//go:build windows
+
+package privileges
+
+// IsPrivileged on windows doesn't matter as we are using connect scan
```

Comment:
The comment must be generic. On windows we need to implement a proper check, or panic otherwise, so the app using this library will be able to detect the issue